src/types.ts

```diff
@@ -8,6 +8,28 @@ export interface TagInfo {
   description: string;
 }
 
+export interface CurveLpTokenInfo extends TokenInfo {
+  extensions: {
+    /** The address of the pool the curve LP token corresponds to, may sometimes
+     * be the address of the token itself  */
+    pool: string;
+
+    /**
+     * The tokens in the pool. Pool assets are assigned an index which can be
+     * used to reference them in the curve contracts.
+     */
+    poolAssets: [string, string] | [string, string, string];
+
+    /** Function signature corresponding to the add liquidity function
+     * on the base token pool contract*/
```

Comment:
minor: is the "base token pool contract" the same as the `pool` address specified here in the extensions? Maybe we can say:

> Function signature corresponding to the add liquidity function on the pool contract

or if it's the lp token then maybe:

> Function signature corresponding to the add liquidity function on the curve lp token contract